deploy/scripts/shared.sh

```
@@ -64,36 +64,23 @@ render_service() {
-                    if [[ ! -r "$container_file" ]]; then
-                        echo "Error: Source container file is not readable: $container_file" >&2
-                        exit 1
-                    fi
-
-                    echo "copy container: ${container_file} -> ${dest_container}"
-                    install -m 644 "$container_file" "${dest_container}"
+        # Process container files except standalone ones
+        for container_file in "${source_dir}/flightctl-${service_name}"/*.container; do
+            if [[ -f "$container_file" ]] &&
+               [[ ! "$container_file" == *"-standalone.container" ]]; then
+                local base_filename=$(basename "$container_file")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)